api/types/src/convert.rs

```diff
@@ -466,7 +466,7 @@ impl<'a, R: MoveResolver + ?Sized> MoveConverter<'a, R> {
                 AbortLocation::Script => format!("Move abort: code {}", abort_code),
             },
             KeptVMStatus::Executed => "Executed successfully".to_owned(),
-            KeptVMStatus::OutOfGas => "Out of gas".to_owned(),
+            KeptVMStatus::OutOfGas => "Gas bid too low".to_owned(),
```

Comment:
`//////// 0L //////// ` should be added at the end of line

Reply:
@MichaelZelbel yes please do this